frontend/src/app/display-gender.pipe.ts

```diff
@@ -1,19 +1,19 @@
-import { Pipe, PipeTransform } from '@angular/core';
+import { Pipe } from '@angular/core';
+import { BaseEnumPipe } from './base-enum-pipe';
+
+export const GENDER_TRANSLATIONS = [
```

Comment:
The keys should be strictly typed (introduce a Gender interface `'male' | 'female' | 'other'`)